src/AlphaVantage/index.js

```diff
@@ -0,0 +1,31 @@
+import * as cryptoDataHandlers from "./CryptoDataHandlers";
+import * as currencyDataHandlers from "./CurrencyDataHandlers";
+import * as stockDataHandlers from "./StockDataHandlers";
+import * as searchTypes from "../features/Search/constants/search-types";
+
+/**
+ * The function fetches data depends on the search-type that was passed
+ *
+ * @param {string} searchType
+ * @param {string} symbol
+ * @returns {Promise<unknown>}
+ */
+export const fetchData = (searchType, symbol) => {
```

Comment:
encapsulating functions that serve the same service/functionality is always easier to read than defining the function at the component level, it keeps stuff simpler